pepper-apis/dsm-core/src/main/resources/liquibase/CMI/PEPPER-445_tissueReceivedEvent.xml

```diff
@@ -5,19 +5,19 @@
         xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
 
-    <changeSet author="simone" id="PEPPER-445_tissueReceivedEvent">
+    <changeSet author="simone" id="PEPPER-445_samplesReceivedEvent">
```

Comment:
Just to sanity check since this is a change to an existing liquibase file--is it safe to assume that this can be applied without triggering a liquibase conflict?

Reply:
yes. liquibase is fine with that

Reply:
I think because the id is changed the hashes are not compared

Reply:
💯